src/App.js

```diff
@@ -4,7 +4,17 @@ import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export class App extends Component {
-  state = {};
+  state = {
+    querry: '',
+  };
+
+  inputAdd = (event) => {
+    if (event.key === 'Enter') {
+      this.setState(
+        { querry: event.target.value },
+      );
```

Comment:
you should save changes and filter movies on every change not only when the user pressed Enter